### PR TITLE
clicking through items during import causes a 'loading' state

### LINKED ISSUE #3729: clicking through items during import should cause a 'loading' state
### Observed behavior

there is currently no user feedback when clicking items in the import UI, and we allow the user to click other things while a request is pending. Note that the URL is changing, but the visible UI is not:

click

### Expected behavior

* When the user clicks an item, we should give some indication that the operation has begun and is pending
* We should also prevent the user from clicking other items

recommend using the widgets that @christianmemije introduced in #3473

### User-facing consequences

Frustration: user might things are broken and make things worse by re-clicking.

### Errors and logs

None

### Steps to reproduce

Navigate through a large channel, such as KA English

### Context

re #3717

### PR DESCRIPTION
**Pull request by jonboiser**
### Summary
Fixes #3729 by flipping on the 'loading' state while the `updateTreeViewTopic` action is fetching data during import/export flows.

Because of how `core-base` works, this still wipes out everything below the nav-bar, but maybe this can be improved later by only hiding certain sections of the UI.

loading

### Reviewer guidance

Test import/export workflow in all combinations and do a lot of navigation.

### References

…

----

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/plugins/device_management/assets/src/state/actions/selectContentActions.js [hunk 1]
```diff
@@ -70,6 +71,7 @@ export function updateTreeViewTopic(store, topic) {
       .then(contents => {
         store.dispatch('SET_CURRENT_TOPIC_NODE', contents);
         store.dispatch('UPDATE_PATH_BREADCRUMBS', topic);
+        store.dispatch('CORE_SET_PAGE_LOADING', false);
```
Comment (indirectlylit) [inline kolibri/plugins/device_management/assets/src/state/actions/selectContentActions.js]: would we also need to set this in the `catch` handler?

**Comment by jonboiser** [comment]
@indirectlylit Updated with a last 'then' to turn off the loading for both 'then' and 'catch' branches.

**Review by indirectlylit — APPROVED** [review]
thanks!

**Comment by rtibbles** [comment]
This was flagged as 0.10, but targeted to develop. @jonboiser could you rebase to release-v0.10.x?

**Pull request merged**